pulsar/consumer_impl.go

```diff
@@ -74,6 +75,11 @@ func newConsumer(client *client, options ConsumerOptions) (Consumer, error) {
 		messageCh = make(chan ConsumerMessage, 10)
 	}
 
+	dlq, err := newDlqRouter(client, options.DLQ)
```

Comment:
what are the defaults here? will every topic by default have dlq enabled?

Reply:
By default the policy is `nil` and the it will result in no-op